src/organizations.js

```diff
@@ -1,12 +1,42 @@
 const { request } = require('./http');
 
 
+const create = (data, opts) => request(opts, {
```

Comment:
I've been wondering about this. We could use something like jsonschema for each resource on the client, but then we would need to update it each time an update is made to the api side.

Only doing validation on the API side means the client doesn't prevent mistakes happening during the dev process, though on the other hand it does avoid the repetition we would need to keep in sync manually, and does keep the client thin.

I'm leaning more to not doing validation on the client side for now, if that is alright?